### ISSUE TITLE
Replace/Remove SVN keywords from API script.

### ISSUE THREAD

**Issue by phloxic** [flowplayer/flash-build#7]
Distributing the script with the following in the license header:

```
 * Date: ${date}
 * Revision: ${revision}
```

[sic] just looks clumsy.

@anssip - do you have something in place to expand these via Makefile:

```
 * Date: @DATE
 * Revision: @REVISION
```

Or should they just be removed?

**Comment by phloxic** [comment]
Both the license headers of
- flowplayer-src.js
- flashembed.js

must be fixed.

**Comment by phloxic** [comment]
Copyright years must be updated.

**Comment by anssip** [comment]
flowplayer-src.js is fixed and pushed to releases.flowplayer.org (copyright year also)

**Comment by phloxic** [comment]
No. flashembed.js + flowplayer-src.js must be concatenated into flowplayer-&lt;version&gt;.js

**Comment by phloxic** [comment]
It's probably a matter of omitting the removal of flowplayer.js before uglification.

**Comment by phloxic** [comment]
flashembed.js still has the svn keywords and other unexpanded vars: https://github.com/flowplayer/flash/blob/master/core/src/javascript/flashembed.js
These _would_ go into the unminified flowplayer.js if it was built. In theory they also should persist in the minified version as flashembed.js could also be used standalone - if jquerytools was still actively maintained ...

**Comment by phloxic** [comment]
Proposal:

``` js
/**
 * Flashembed - New wave Flash embedding
 *
 * NO COPYRIGHTS OR LICENSES. DO WHAT YOU LIKE.
 */
```

for the non-minified version - removed automatically in the minified version.

**Comment by phloxic** [comment]
@anssip - the copyright year is still 2008-2011 in flowplayer-src.js

**Comment by phloxic** [comment]
@anssip - now _all_ version information is removed from the scripts.
It would be better if the build scripted added the release version and release date - as it is done for the fp html5 api.
